jetty-http/src/main/java/org/eclipse/jetty/http/HttpURI.java

```diff
@@ -77,6 +103,8 @@
     String _uri;
     String _decodedPath;
 
+    boolean _ambiguousSegment;
```

Comment:
Method `clear()` does not null-out field `_user`. Perhaps move `clear()` close to the fields?